mappings/net/minecraft/structure/GrassFeature.mapping

```diff
@@ -1,6 +1,4 @@
-CLASS net/minecraft/class_477 net/minecraft/structure/GrassFeature
-	FIELD field_1352 id I
-	FIELD field_1353 blockId I
-	METHOD <init> (II)V
-		ARG 1 id
-		ARG 2 blockId
+CLASS net/minecraft/class_959 net/minecraft/structure/GrassFeature
```

Comment:
any idea why the intermediary class name changed?

Reply:
I changed the original GrassFeature to be PlantFeature iirc

Reply:
git is just being weird

Reply:
It's now MetaPlantFeature (plant feature + metadata) @Blayyke

Reply:
looks good to me then